index.html

```diff
@@ -57,16 +73,95 @@ <h3>Multi-Post Stories</h3>
               </ul>
               <button>See Project</button>
           </div>
-    
-        <div class="mobile-cards">
-          <img src="images/Works-image.png" alt="card-image" class="Cards-Bunch">
-          <img src="images/Works-image.png" alt="card-image" class="Cards-Bunch">
-          <img src="images/Works-image.png" alt="card-image" class="Cards-Bunch">
-          <img src="images/Works-image.png" alt="card-image" class="Cards-Bunch">
-          <img src="images/Works-image.png" alt="card-image" class="Cards-Bunch">
-          <img src="images/Works-image.png" alt="card-image" class="Cards-Bunch">
+
+  <div class="multi-cards">
+      <div class="all-card">
+          <div class="card">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                  <li>HTML</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+                <button>See project</button>
+            </div>
+        </div>
+         <div class="all-card">
+          <div class="card">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                  <li>HTML</li>
+                  <li>Bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+                <button>See project</button>
```

Comment:
- [ ] Nice job implementing the cards 👏 . However, here are a few things to work on:
- Your cards are missing the button provided on Figma
- The white background and button you see is an interaction that should happen on all of your cards on hover. **Kindly make such features on all cards**

## View:
![image](https://user-images.githubusercontent.com/50990925/228418464-70306648-a083-474b-a855-7def8b8548e2.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/228418520-abe02f87-86a8-480c-bfff-80ee2f005a0c.png)

Reply:
Hey I am not sure what you mean by all the cards must have the buttons, I only see one button that says see projects. Also can you please clarify where on the requirements it says the first card is a hover? Kindly get back to me about this information.